### ISSUE TITLE
[FE][Accessibility] Update the heading for the in progress position 

### ISSUE THREAD

**Issue by mdewey** [department-of-veterans-affairs/va.gov-team#26166]
## User Story

As a veteran, I should have an accessible site so that I can navigate the page with ease, regardless of the level of accessibility 

## Related Documentation

- [Original Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24641)
- [The comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/24641#issuecomment-856253409)

## Description

Update the `Your questionnaire is in progress` to be outside of the green box. Work with platform and accessibility on how widespread this change should be

## Tasks

> this is a list of technical tasks that need for this ticket

- [ ] Update the in progress screen. 
- [ ] Create user tests
- [ ] Create e2e tests
- [ ] Update relevant documentations

## Acceptance Criteria

> when is the ticket done
- [ ] Passes Accessibility checks using axe
- [ ] Deployed to staging.
- [ ] Designer has signed off on the feature
- [ ] Accessibility has signed off on the feature.

**Comment by julieadair1** [comment]
@dillo , @mdewey - is this something that still needs to be addressed?
cc: @jparker0425
